webui/react/src/hooks/useModal/Experiment/useModalExperimentMove.tsx

```diff
@@ -292,7 +252,6 @@ const useModalExperimentMove = ({ onClose }: Props): ModalHooks => {
     }
   }, [
     closeNotification,
-    destSettings.projectId,
```

Comment:
nit: I think `projectId` does need to be in this dep array since you don't want the callback function to be calculated using an old value.